helm/ctdapp/templates/prometheus-rule-prod.yaml

```
@@ -0,0 +1,145 @@
+  groups:
+    - name: peers
+      rules:
+        - alert: PeersEligibleNodesMonotonicity
+          annotations:
+            description: >-
+              Eligible nodes count should both increase and decrease
+            summary: Eligible nodes count too monotonic
+          expr:
+          for: 10m
+          labels:
+            severity: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)